dll/win32/msi/dialog.c

```diff
@@ -3157,8 +3157,17 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
 
         /* the width must be a positive number
          * if a width is invalid, all remaining columns are hidden
+         * Skip in case of prefix the string of displayed characters with {\style} or {&style}.
          */
         if ( !wcsncmp( num, L"-", 1 ) || !str_is_number( num ) ) {
+#ifdef __REACTOS__
+            if (count == 0 && (!wcsncmp( num, L"\\", 1 ) || !wcsncmp( num, L"&", 1 )))
```

Comment:
no whiteaspace padding at innerside of braces. Use
`if (count == 0 && (!wcsncmp(num, L"\\", 1) || !wcsncmp(num, L"&", 1)))`